plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -97,6 +98,13 @@ algorithm layered(LayeredLayoutProvider) {
     supports edgeRouting.splines.mode
     supports edgeRouting.splines.sloppy.layerSpacingFactor
     
+    // topdown layout
```

Comment:
Why does elk.force not need these options?

Reply:
I think force, mrtree and fixedlayout do need them